oak_idl/src/lib.rs

```diff
@@ -114,7 +118,7 @@ impl Transport for Channel {
         let request_bytes =
             bincode::serialize(&request_message).map_err(|_| TransportError::InvalidMessage)?;
```

Comment:
Is the plan to convert this to flatbuffers later, or will these be deleted later, or maintained in parallel?